CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [Unreleased 5.2.6]
 ## Feature
 - Workflow : Add support to select real status fields.
+- STOCK CONFIG : add three fields in stock move printing setting
```

Comment:
which fields ? Please, in changelog file, describe the feature as much you can. A person should undestand the feature just reading this file.